### PR TITLE
Release notes to support 0.21.0

### PR DESCRIPTION
**Pull request by SueChaplain**
Changes to the OpenJ9 VM  for release 0.21.0

[ci skip]

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@knn-k Hi - please can you check the list of known issues and limitations for ARM64 and let me know if any can be removed for 0.21.0

**Comment by knn-k** [comment]
For AArch64 Linux,

- #8034 (Balanced GC) is the only remaining restriction.
- Concurrent scavenge mode is supported.
- JIT DLT and recompilation are enabled.
- AOT is used by default.
- #9251 was closed.

There are more performance features to be implemented in the JIT, but I don't think they need to be listed as "Known Issues".

**Comment by SueChaplain** [comment]
Thanks @knn-k - release notes updated.

File: doc/release-notes/0.21/0.21.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/9610">#9610</a></td>
+<td valign="top">New option that affects the handling of the SIGABRT signal</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">By default, the OpenJ9 VM handles SIGABRT signals and generates dump files. A new option is available to force OpenJ9 to match the
+reference implementation, where the signal is handled by the default operating system handler. To match the reference implementation, set <tt>-XX:+HandleSIGABRT</tt>.</td>
```
Comment (pshipton) [inline doc/release-notes/0.21/0.21.md:62]: To match the reference, it's `-` not `+`.

Comment (SueChaplain) [reply doc/release-notes/0.21/0.21.md]: Fixed.

**Pull request merged**